tests/helpers/test_script.py

```diff
@@ -85,21 +41,25 @@ def check_action():
     return flag
 
 
-async def test_firing_event_basic(hass):
+async def test_firing_event_basic(hass, caplog):
     """Test the firing of events."""
     event = "test_event"
     context = Context()
     events = async_capture_events(hass, event)
 
     sequence = cv.SCRIPT_SCHEMA({"event": event, "event_data": {"hello": "world"}})
-    script_obj = script.Script(hass, sequence)
+    script_obj = script.Script(
+        hass, sequence, "Test Name", "test_domain", running_description="test script"
+    )
 
     await script_obj.async_run(context=context)
     await hass.async_block_till_done()
 
     assert len(events) == 1
     assert events[0].context is context
     assert events[0].data.get("hello") == "world"
+    assert ".test_name:" in caplog.text
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)